lessons/2_more_strings_and_functions/lesson.py

```diff
@@ -113,6 +113,15 @@ def my_cool_function(pet_name):
 # # # MH guess: (7*3)-5=16
 # # ANSWER A CODE GOES HERE
 print("the answer to exercise 1A is:",multiply_and_subtract_five(7,3))
+print("the answer to exercise 1A is: {}.".format(multiply_and_subtract_five(7,3)))
+
+name = "Emily"
+food = "tacos"
+print("{} would like to eat some {}".format(name, food))
+
+print("{name} would like to eat some {food}.")
+# # # MH why isn't this f-string working?
```

Comment:
there is no f!

It should be

print(f"{name} would like to eat some {food}.")